### PR TITLE
Feature/change request permission route

### PR DESCRIPTION
**Pull request by cristiantela**

### CODE REVIEW

File: src/services/SecurityService.js [hunk 1]
```diff
@@ -34,6 +35,7 @@ Oidc.Log.level = Oidc.Log.INFO;
 mgr.events.addUserLoaded(function (user) {
   console.log('New User Loaded：', arguments);
   console.log('acess_token: ', user.access_token)
+  ApiInstance.defaults.headers.common['Authorization'] = 'Bearer ' + user.access_token;
```
Comment (matheusm) [inline src/services/SecurityService.js:38]: I think you can use string literals

File: src/router.js [hunk 2]
```diff
@@ -150,7 +150,7 @@ router.beforeEach((to, from, next) => {
   if (requiresAuth) {
       SecurityService.getUser().then(
         success => {
-          axios.defaults.headers.common['Authorization'] = 'Bearer ' + success.access_token
+          ApiInstance.defaults.headers.common['Authorization'] = 'Bearer ' + success.access_token
```
Comment (matheusm) [inline src/router.js:153]: String literals here too

File: src/components/orgs/UserManagement.vue [hunk 3]
```diff
@@ -313,9 +340,8 @@ export default {
         this.$emit('users', this.users.concat(addedUser));
 
         this.userSearch = '';
-        this.role = '3';
-      } catch (e) {
-        this.users = [];
+      } catch (error) {
+        console.log(error);
```
Comment (matheusm) [inline src/components/orgs/UserManagement.vue:344]: I think you can remove log or handle with error

File: src/components/orgs/UserManagement.vue [hunk 4]
```diff
@@ -269,40 +282,54 @@ export default {
...
 
-      try {
-        const email = this.userSearch.toLowerCase();
+      const email = this.userSearch.toLowerCase();
 
-        if (this.users.find((user) => user.email === email)) {
-          this.userError = this.alreadyAddedText;
-          return false;
-        }
+      if (!/^\w+([-+.']\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*$/.test(email)) {
```
Comment (matheusm) [inline src/components/orgs/UserManagement.vue:287]: Create an util to help with masks

**Review by matheusm — APPROVED** [review]

File: src/components/orgs/UserManagement.vue (hunk 4 above)
Comment (cristiantela) [reply src/components/orgs/UserManagement.vue]: I created temporally a object named rules, for future we think more about how we could centralize those rules.

**Pull request merged**